grafana/risingwave-dashboard.dashboard.py

```diff
@@ -480,6 +480,7 @@ def metric(name, filter=None):
 
 def quantile(f, percentiles):
     quantile_map = {
+        "60": ["0.6", "60"],
```

Comment:
cool, i have to questions
1. Don't we need p999 anymore?
2. Can we extend this fine-grained bucket statistics to other metrics, such as read/write duration

Reply:
P999 and max is duplicated. I think we only need to keep max.
In fact, if I do not close max and p999, I could even not see the value of p60 and p90....

Reply:
Yes, I also often encounter this problem. This pr only modifies `create_iter_time`. Is there any update?

Reply:
I would change it rollback. But I still do not think we need P999 and max.